docs/api/remote.md

```diff
@@ -1,29 +1,29 @@
 # remote
 
 The `remote` module provides a simple way to do inter-process communication
-between the renderer process and the main process.
+between the renderer process (web page) and the main process.
 
 In Electron, only GUI-unrelated modules are available in the renderer process.
-Without the `remote` module, users who wanted to call a main process API in
+Without the `remote` module, users who want to call a main process API in
 the renderer process would have to explicitly send inter-process messages
-to the main process. With the `remote` module, users can invoke methods of
-main process object without explicitly sending inter-process messages,
-similar to Java's
-[RMI](http://en.wikipedia.org/wiki/Java_remote_method_invocation).
+to the main process. With the `remote` module, users can invoke methods of the
```

Comment:
"you" though I think in most places I just was on the look out for use of "one" so there may be a few of these. I'll look for them more now.